client/web/compose/src/components/Namespaces/Reminders/Edit.vue

```diff
@@ -120,13 +120,13 @@
         {{ $t('label.back') }}
       </b-button>
 
-      <b-button
+      <c-button-submit
         data-test-id="button-save"
-        variant="primary"
-        @click="$emit('save', reminder)"
-      >
-        {{ $t('label.save') }}
-      </b-button>
+        :disabled="disableSave"
+        :processing="processingSave"
```

Comment:
I don't see the need for disabled here, since processing should disable the button anyway